### DISCUSSION TITLE
EIP-4844 Support: Type3 Transactions

### DISCUSSION THREAD

**Discussion by 0xBreadguy** [rainbow-me/rainbowkit#1862] [Ideas]
Support for blob-carrying transactions introduced in EIP-4844. Currently the only way to enable this function for users is by having them sign a non-human-readable transaction.

**Comment by 0xBreadguy** [comment]
Closing for alternative

**Comment by 0xmkh** [comment]
@0xBreadguy Our team will look closely into EIP-4844 support 🙏

**Reply by 0xBreadguy** [reply]
Thanks @magiziz - would love to open up a channel to discuss this if possible. We're working on something and would like to have end users get all of the niceties they're used to with traditional transactions (simulation, gas costs, etc).

**Reply by 0xmkh** [reply]
Sounds good. Yeah let me see what i can do on my end here 🙏
